tests/test_driver.py

```diff
@@ -19,6 +19,15 @@ class TestBigchainDB:
                                       'http://node-2:9984')),
         (('node-1', 'node-2'), {'app_id': 'id'}, ('http://node-1:9984',
                                                   'http://node-2:9984')),
+        (({'endpoint': 'node-1',
+           'headers': {'app_id': 'id'}},
+          {'endpoint': 'node-2',
+           'headers': {'app_id': 'id'}}),
+            None,
+            ({'endpoint': 'http://node-1:9984',
+              'headers': {'app_id': 'id'}},
+             {'endpoint': 'http://node-2:9984',
+              'headers': {'app_id': 'id'}})),
```

Comment:
would it make sense to just remove line 18/19 because it is more or less the same set of nodes?